JUnit version 4.13.1
....E
Time: 0.55
There was 1 failure:
1) test1(io.restassured.internal.matcher.xml.LoadFromClasspathSupport_ESTest)
java.lang.AssertionError: expected null, but was:<java.io.ByteArrayInputStream@4953bfec>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotNull(Assert.java:756)
	at org.junit.Assert.assertNull(Assert.java:738)
	at org.junit.Assert.assertNull(Assert.java:748)
	at io.restassured.internal.matcher.xml.LoadFromClasspathSupport_ESTest.test1(LoadFromClasspathSupport_ESTest.java:37)

FAILURES!!!
Tests run: 4,  Failures: 1

